portal-impl/src/com/liferay/portal/service/impl/LayoutLocalServiceHelper.java

```diff
@@ -513,6 +515,17 @@ public void validateFriendlyURLs(
 
 		LayoutFriendlyURLsException layoutFriendlyURLsException = null;
 
+		Set<String> friendlyURLSet = new HashSet<>(friendlyURLMap.values());
+
+		if (friendlyURLSet.size() != friendlyURLMap.size()) {
+			LayoutFriendlyURLException layoutFriendlyURLException =
+				new LayoutFriendlyURLException(
+					LayoutFriendlyURLException.POSSIBLE_DUPLICATE);
```

Comment:
I believe this should be DUPLICATE rather than POSSIBLE_DUPLICATE.

POSSIBLE_DUPLICATE seems to be reserved for cases where the Friendly URL could be a duplicate of another page's non-friendly URL (such as if the friendly URL consists only of numeric characters that could cause it to conflict with a page's URL that was constructed with it's layoutId).

DUPLICATE is used for cases where we know that there are two identical friendly URLs (which matches the case here).